docs_source_files/config.toml

```diff
@@ -167,3 +167,30 @@ name = "<i class='fab fa-fw fa-github'></i> GitHub repo"
 identifier = "ds"
 url = "https://github.com/SeleniumHQ/docs"
 weight = 10
+
+[Languages.ja]
+title = "セレニウムドキュメント"
+weight = 1
```

Comment:
Could you please change the weight to `6`? So it follows the current order.

Reply:
Updated the weight value based on the review. 
I also noticed when referencing Selenium as a browser automation tool, "Selenium" is used instead of セレニウム. I change the title to reflect this.